www/content/docs/pages.mdx

```diff
@@ -168,15 +134,20 @@ export async function getStaticProps(context) {
     params.addFields("node--article", ["title", "path", "body", "uid"])
   }
 
-  const node = await drupal.getResourceFromContext(type, context, {
+  const node = await drupal.getResourceByPath(path, {
```

Comment:
Since we have already translated the path here, sing getResource could save us an API call (depending on how things are being cached)

```suggestion
  const node = await drupal.getResource(type, path.entity.uuid {
```